Automattic/woocommerce-payments
# Fix e2e tests related to the WooCommerce blocks' checkout

Fixes #3274 

#### Changes proposed in this Pull Request

Update the flow for filling up the WCB's checkout in the e2e tests

* Fill up the billing fields instead of the shipping ones
* Fill up the state field

Waiting for the tests from this PR run to finish before requesting a review.

#### Testing instructions

* Check out the 'develop' branch
* Run npm run test:e2e ./tests/e2e/specs/shopper/shopper-wc-blocks-checkout-failures.spec.js or npm run test:e2e ./tests/e2e/specs/shopper/shopper-wc-blocks-checkout-purchase.spec.js
* Opposite to the related issue, all tests should pass now

Or just make sure those e2e tests from this PR pass, while they're failing in other PRs.

There are other tests failing, but I kept them out of the scope of this PR.

-------------------

- [ ] Added changelog entry (or does not apply) - Would this need a changelog entry?

**Post merge**
